src/annotator/guest.js

```diff
@@ -615,4 +631,14 @@ export default class Guest {
   get sideBySideActive() {
     return this._sideBySideActive;
   }
+
+  /**
+   * Return the tags of annotations that are currently displayed in a focused
+   * state.
+   *
+   * @return {Set<string>}
+   */
+  get focusedAnnotationTags() {
```

Comment:
I am lazy. Is this externally used (got) anywhere, or is this currently a test affordance?

Reply:
This is a test affordance.